lib/measured/rails/active_record.rb

```diff
@@ -1,9 +1,71 @@
-module Measured::Rails
+module Measured::Rails::ActiveRecord
   extend ActiveSupport::Concern
 
   module ClassMethods
-    def measured
+    def measured(measured_class, *fields)
+      options = fields.extract_options!
+      options = {}.merge(options)
```

Comment:
is this not redundant? extract_options! returns a blank hash if there's nothing, doesn't it?